client/constants/review-constants.ts

```diff
@@ -163,14 +163,15 @@ export const NETWORK_REVIEW_QUESTIONS = [
       '1 point = $100-200',
       '0 points = >$200',
     ],
+    tooltiptext: 'Divide Total Estimated Project Cost by Community Population',
   },
   {
     maxScore: 4,
     label: 'Section 3. Components score (auto)',
     name: 's3ComponentsScore',
     secondaryList: [
-      `1 point each for a 'yes' answer to questions 3-5 (automated)`,
-      '1 point for at least 2 boxes checked on question 6. Up to 4 points total',
+      `1 point each for a 'yes' answer to questions 6-8 (automated)`,
```

Comment:
Does it need to be updated in `helpers` as well if it's automated?

Reply:
I updated the comments in it, but no the helper doesn't care about question #